File: 백준/Bronze/10409. 서버/서버.c
#include <stdio.h>
int main(void) {
    int n, t, elem, cnt = 0, rlt = 0;
    scanf("%d %d", &n, &t);
    for (int i = 0; i < n; i++) {
        scanf("%d", &elem);
        if (rlt + elem > t)
            break;
        rlt += elem;
        cnt += 1;
    }
    printf("%d", cnt);
    return 0;
}

File: 백준/Bronze/10409. 서버/서버.py
import sys
input=lambda:sys.stdin.readline().rstrip()
n,t=map(int,input().split())
arr=[*map(int,input().split())]
rlt,cnt=0,0
for i in range(n):
    if rlt+arr[i]>t: break
    cnt+=1
    rlt+=arr[i]
print(cnt)